### PR TITLE
API Updates

### PR DESCRIPTION
**Pull request by richardm-stripe**
Codegen for openapi 403fdb8.
r? @richardm-stripe
cc @stripe/api-libraries

## Changelog
* Added support for new resource `BillingPortal.Configuration`
* Added support for `configuration` and `on_behalf_of` on `Session#create` and `BillingPortal.Session`
* `WebhookEndpoint#create.enabled_events[]` and `WebhookEndpoint#update.enabled_events[]` added new enum members: `billing_portal.configuration.created and billing_portal.configuration.updated`

### CODE REVIEW

File: src/main/java/com/stripe/model/billingportal/Configuration.java [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+       *
+       * <p>One of {@code always_invoice}, {@code create_prorations}, or {@code none}.
+       */
+      @SerializedName("proration_behavior")
+      String prorationBehavior;
+
+      @Getter
+      @Setter
+      @EqualsAndHashCode(callSuper = false)
+      public static class SubscriptionUpdateProduct extends StripeObject {
```
Comment (remi-stripe) [inline src/main/java/com/stripe/model/billingportal/Configuration.java:336]: This one should be named `Product` so the client_metadata internally is wrong

File: src/test/java/com/stripe/functional/billingportal/ConfigurationTest.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+import com.stripe.param.billingportal.ConfigurationCreateParams;
+import com.stripe.param.billingportal.ConfigurationUpdateParams;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+public class ConfigurationTest extends BaseStripeTest {
+  public static final String SESSION_ID = "bps_123";
```
Comment (remi-stripe) [inline src/test/java/com/stripe/functional/billingportal/ConfigurationTest.java:19]: bpc_123

File: src/test/resources/api_fixtures/billingportal/configuration.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (remi-stripe) [inline src/test/resources/api_fixtures/billingportal/configuration.json:1]: We shouldn't add this. stripe-mock should have the right fixtures instead so we should remove and fix the test above

**Review by remi-stripe — CHANGES_REQUESTED** [review]

File: src/test/java/com/stripe/functional/billingportal/ConfigurationTest.java (hunk 2 above)
Comment (richardm-stripe) [inline src/test/java/com/stripe/functional/billingportal/ConfigurationTest.java:19]:
```suggestion
  public static final String SESSION_ID = "bpc_123";
```

**Comment by richardm-stripe** [comment]
r? @remi-stripe
* Removed extra test
* SubscriptionUpdateProduct is now just Product per [latest openapi](https://github.com/stripe/openapi/commit/8887a20bdb70c5145e1e29e746e4cf7874b84997)

**Review by remi-stripe — APPROVED** [review]

**Pull request merged**